backend/satellite_tools/spacewalk-extra-http-headers

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Is the shebang changed when building the package? If not, I fear that using `env` is not allowed and we do not want python2 anymore.

Reply:
Yep, the shebang is changed when building `spacewalk-backend-tools`: https://github.com/uyuni-project/uyuni/blob/master/backend/spacewalk-backend.spec#L279

Reply:
ok